tcf_website/management/commands/load_review_drive.py

```diff
@@ -71,6 +71,12 @@ def handle(self, *args, **options):
             print("ERROR: Semester not found")
             return
 
+        if settings.REVIEW_DRIVE_ID is None or \
```

Comment:
`if None in [settings.REVIEW_DRIVE_ID, settings.REVIEW_DRIVE_PASSWORD, settings.REVIEW_DRIVE_EMAIL]`